source/client-side-encryption/tests/mapReduce.template.yml

```diff
@@ -0,0 +1,26 @@
+runOn:
+  - minServerVersion: "4.1.10"
+database_name: &database_name "default"
+collection_name: &collection_name "default"
+
+data:
+  - &doc0_encrypted { _id: 1, x: 1, ssn: {{ciphertext("457-55-5462", field="ssn")}} }
+  - &doc1_encrypted { _id: 2, x: 2, ssn: {{ciphertext("123-45-6789", field="ssn")}} }
+json_schema: {{schema()}}
+key_vault_data: [{{key()}}]
+
+tests:
+  - description: "mapReduce deterministic encryption"
+    skipReason: "TODO: throw on unsupported commands in Java PoC"
```

Comment:
Let's remove this and i'll fix the Java PoC

Reply:
Done.